docs/source/conf.py

```diff
@@ -181,6 +181,7 @@ def setup(app: Sphinx) -> None:
 # See https://sphinx-codeautolink.readthedocs.io/en/latest/reference.html#configuration
 codeautolink_autodoc_inject = False
 codeautolink_global_preface = """
+import trio
 from trio import *
 """
```

Comment:
most of the code uses `trio.xxxx`, maybe even all of it, but not much reason not to keep `from trio import *`. Although it's plausible we should also import `testing`/`lowlevel`/etc, have not scoured the docs that thoroughly yet.